### DISCUSSION TITLE
loading Table page dynamically

### DISCUSSION THREAD

**Discussion by merlinarer** [zauberzeug/nicegui#2735] [Q&A]
### Question

Hello, I have a very long table (each row may have long content) to render and I tried to use ui.table (aggrid also) but found it  stuck sometimes. I noticed that qtable offers a dynamical loading [here](https://quasar.dev/vue-components/table#server-side-pagination-filter-and-sorting) which loads only a Table page once (5 rows for example if rowsPerPage=5). Is there a way to do the same thing with nicegui?

**Comment by falkoschindler** [comment]
Hi @merlinarer,

Server-side pagination is currently not supported, but is being discussed in feature request https://github.com/zauberzeug/nicegui/discussions/2351.
